desktop/TransactionService.ts

```diff
@@ -17,8 +17,9 @@ class TransactionService extends NetServiceFactory {
         if (error) {
           logger.error('grpc SubmitTransaction', error);
           resolve({ error });
+        } else {
+          resolve({ error: null, txstate: response.response.txstate });
```

Comment:
Non-consistent return. It may cause problems.

Reply:
this part is actually WIP, will be the main focus of next PR